examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -68,8 +68,11 @@ class FollowStatePolicy(Enum):
 START_FETCH_ROBOT_DIS_THRESHOLD = (
     1.8  # if the human is too close to Spot, they block Spot
 )
+# The distance between the robot and the human to switch from point nav to social nav
 FOLLOW_SWITCH_GEO_DIS_FOR_POINT_SOCIAL_NAV = 2.5
 PICK_STEPS = 40
+# The distance between the robot and the human to consider termination of FOLLOW state
+ROBOT_CLOSE_TO_HUMAN_DIS = 2.0
```

Comment:
Ruta will need a way to disable this for the social-nav study, since we want "pure learned social-nav" with no extra heuristics. Can you add a command-line arg or similar?

Reply:
Thanks, Eric! This has been done. Please see the snapshot document, and let me know if you have any questions!

Reply:
you can do  `--enable-hybrid-social-nav` in CLI to enable hybrid social nav